packages/sharec-core/lib/steps/readConfigs.js

```diff
@@ -18,12 +19,34 @@ const readConfigs = async (input) => {
   try {
     const configsPaths = await find(configsPath, '**/*')
     const withoutLocks = configsPaths.filter((config) => !/(\.|-)lock/.test(config))
+
+    const withoutBinaries = []
+
+    for (const file of withoutLocks) {
+      if(isText(file) || getEncoding(await readBuffer(file, 24)) === 'utf8') {
+        withoutBinaries.push(file)
+      } else {
+        input.binaries.push({
+          filename: file.replace(configsPath, '').replace(/^\//, ''),
+          original: file
```

Comment:
There is not necessary to store file source. Just copy upcoming ones by stored paths in `writeBinaries` step